ddtrace/mocktracer/mockspan.go

```diff
@@ -102,6 +105,7 @@ type mockspan struct {
 	tags         map[string]interface{}
 	finishTime   time.Time
 	finished     bool
+	source       string
```

Comment:
maybe better to call this either `component` or `integration` rather than introducing a new name?